src/laboratory/api/shelfobject.py

```diff
@@ -332,6 +335,11 @@ def delete(self, request, org_pk, lab_pk, **kwargs):
         :return:
         """
         self._check_permission_on_laboratory(request, org_pk, lab_pk, "delete")
+        serializer = ShelfObjectDeleteSerializer(data=request.data, laboratory=self.laboratory)
+        serializer.is_valid(raise_exception=True)
+        utils.organilab_logentry(self.request.user, serializer.validated_data['shelfobj'], DELETION, relobj=self.laboratory)
+        serializer.validated_data['shelfobj'].delete()
+        return Response({'message': _('The item was deleted successfully')}, status=200)
```

Comment:
The keyword should be "detail" instead of "message" to follow what we agreed on.
Also, it should be a JsonResponse instead of a Response.

Reply:
Corrected